### DISCUSSION TITLE
能不能批量复制图片的链接地址

### DISCUSSION THREAD

**Discussion by shaojiuli** [lsky-org/lsky-pro#370] [想法]
如题

**Comment by 0xxb** [comment]
下个版本前看看怎么弄，手机端不太好适配。

**Comment by shaojiuli** [comment]
感谢感谢

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

________________________________
发件人: Wisp X ***@***.***>
发送时间: Friday, April 1, 2022 5:33:50 PM
收件人: lsky-org/lsky-pro ***@***.***>
抄送: shaojiuli ***@***.***>; Author ***@***.***>
主题: Re: [lsky-org/lsky-pro] 能不能批量复制图片的链接地址 (Discussion #370)


下个版本前看看怎么弄，手机端不太好适配。

―
